msgraph-developer-proxy/ProxyHost.cs

```diff
@@ -11,6 +11,43 @@ internal class ProxyHost {
     private Option<int?> _portOption;
     private Option<LogLevel?> _logLevelOption;
     private Option<bool?> _recordOption;
+    private static Option<string?>? _configFileOption;
+
+    private static bool _configFileResolved = false;
+    private static string _configFile = "appsettings.json";
+    public static string ConfigFile {
```

Comment:
Implemented this as a static with custom option parsing, because the value of this option needs to be available before we invoke the RootCommand which is typically responsible for parsing args.